### DISCUSSION TITLE
Using Interchange to convert an `openmm.System` to an Amber PRMTOP file

### DISCUSSION THREAD

**Discussion by mattwthompson** [openforcefield/discussions#35] [Infrastructure]
Stemming from https://github.com/openforcefield/openff-interchange/issues/868#issuecomment-1878822464

@hjuinj Is this using a SMIRNOFF force field or a force field provided by OpenMM (possibly an Amber force field)?

**Comment by hjuinj** [comment]
Thanks Matt, this would be prot-lig complex with Amber protein forcefield and openff for the ligand

**Reply by hjuinj** [reply]
Sorry @mattwthompson, just tagging you in case no notification was sent you way, as you said you might already have a workaround.

**Reply by mattwthompson** [reply]
With an Amber force field (other than ff14SB) I can't advertise any tricks. Any of these examples, some of which you may already have seen, would be starting points to consider: https://github.com/openforcefield/openff-toolkit/pull/1805
